src/sentry/utils/metrics.py

```diff
@@ -78,7 +75,8 @@ def worker():
 
         self._started = True
 
-    def incr(self, key, instance=None, tags=None, amount=1):
+    def incr(self, key, instance=None, tags=None, amount=1,
+             sample_rate=settings.SENTRY_METRICS_SAMPLE_RATE):
         if not self._started:
             self._start()
         self.q.put((key, instance, tags, amount))
```

Comment:
You need to pass `sample_rate` here too, otherwise, it’ll error when unpacking the tuple on L58.

Reply:
Completed via https://github.com/getsentry/sentry/pull/13431/commits/26672a734f4247dec76b6b942f04609e979f4707